.github/CODEOWNERS

```
@@ -1,4 +1,2 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
-* @damien-urruty-sonarsource @anita-stanisz-sonarsource @johann-beleites-sonarsource @leo-geoffroy-sonarsource
-# The owner team ; keep this at the end of the file
-.github/CODEOWNERS @sonarsource/sonarqube-team
+* @sonarsource/analysis-experience-squad
```

```suggestion
.github/CODEOWNERS @sonarsource/analysis-experience-squad
```

Otherwise everyone in the squad will get pinged on every PR